features/environment.py

```diff
@@ -0,0 +1,25 @@
+from behave import use_fixture
+from behave.fixture import use_fixture_by_tag, use_fixture
+from fixtures import graphql_client, database_seed, stellar_snapshot, login, signup, password_reset, shops_fixtures
+
+
+fixtures_registry = {
+    'fixture.login': login,
+    'fixture.signup': signup,
+    'fixture.password-reset': password_reset,
+    'fixture.shops': shops_fixtures
+}
+
+
+def before_tag(context, tag):
+    if tag.startswith("fixture."):
+        return use_fixture_by_tag(tag, context, fixtures_registry)
+
+
+def before_all(context):
+    use_fixture(graphql_client, context)
```

Comment:
This initializes the `graphql_client` once and for all.

Reply:
Is this file a "magic" name `environment.py` that has a special meaning for `behave` ? It is not imported anywhere

Reply:
yes it is; with `behave`, there's a lot of magic